styles.css

```diff
@@ -1,3 +1,69 @@
-.main-color{
+*{ 
+    -moz-box-sizing: border-box; 
+    -webkit-box-sizing: border-box; 
+     box-sizing: border-box !important;
```

Comment:
I would suggest to remove !important as it is not needed

Reply:
DONE